viral/rastermap_utils.py

```
@@ -16,7 +16,7 @@
     degrees_to_cm,
     get_wheel_circumference_from_rig,
 )
-from viral.models import Cached2pSession, ImagedTrialInfo
+from viral.models import Cached2pSession, TrialInfo, ImagedTrialInfo
```

Ruff (0.8.2)

Redefinition of unused `TrialInfo` from line 14

(F811)